src/main/java/com/epam/esm/entity/Order.java

```diff
@@ -0,0 +1,103 @@
+package com.epam.esm.entity;
+
+import com.epam.esm.converter.ZonedDateTimeAttributeConverter;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import javax.persistence.*;
+import java.math.BigDecimal;
+import java.time.ZonedDateTime;
+import java.util.Objects;
+
+@Entity(name = "Order")
+@Table(name = "orders")
+@JsonIgnoreProperties({"user"})
+public class Order implements Identifiable {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private long id;
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name="user_id")
+    private User user;
+    @OneToOne(fetch = FetchType.LAZY)
```

Comment:
Why `LAZY`?

Reply:
@a1exanddrovich please read more about the lazy initialization and think once again where do you really need it